articles/data-binding-with-angular/index.md

```diff
@@ -0,0 +1,301 @@
+**Angular** is a platform and design framework for building a SPA using HTML and TypeScript.
+ Angular is written in TypeScript(although the first version, i.e.AngularJS, was written in JS).
+ It implements its core and optional functionalities as a set of TypeScript libraries that 
+ you import into your apps.
+
+Before diving into the tutorial, we will have a look at
```

Comment:
Lovely! This is great! 
It's always recommended to add a prerequisites area to inform your reader of what is needed beforehand to fully understand the topics you will be exploring.